### PR TITLE
worked on correct writing

### PR DESCRIPTION
**Pull request by Seid31**
https://ref.eteam.work/index.php?action=action_list&entity_id=32

### CODE REVIEW

**Review by PeterKuzyk — APPROVED** [review]

File: src/test/java/EntityPlatformFunctionsTest.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
+import org.testng.annotations.Ignore;
```
Comment (Stan575) [inline src/test/java/EntityPlatformFunctionsTest.java:6]: Seems to have no impact

File: src/test/java/EntityPlatformFunctionsTest.java [hunk 2]
```diff
@@ -95,6 +96,7 @@ public void functionCancelTest() {
         Assert.assertTrue(constant_table.getText().isEmpty());
     }
 
+
```
Comment (Stan575) [inline src/test/java/EntityPlatformFunctionsTest.java:99]: Extra line?

File: src/test/java/EntityPlatformFunctionsTest.java (hunk 1 above)
Comment (Seid31) [reply src/test/java/EntityPlatformFunctionsTest.java:6]: Yes, need to remove I used it before but forgot to remove it

**Review by ArinaJur — APPROVED** [review]

**Review by Stan575 — APPROVED** [review]
Good job Seid!

**Review by Seid31 — COMMENTED** [review]
> Good job Seid!

Thank you, Stan!

**Pull request merged**